packages/ColorPicker/src/ColorPicker.tsx

```diff
@@ -47,15 +181,36 @@ export interface ColorPickerProps {
 }
 
 const ColorPicker: React.FunctionComponent<ColorPickerProps> = ({ 
+    brand: propBrand,
     className, 
     dataTest,
     disabled = false,
     onSelect,
-    selectedColor = "dandelion200"
+    selectedColor: propSelectedColor
 }: ColorPickerProps) => {
+    const brand = propBrand ?? document.documentElement.getAttribute("data-brand") ?? "igloo";
+    
+    const getDefaultColor = (): ColorName => {
+        return brand === "workleap" ? "decorativeOption3" : "dandelion200";
+    };
+
+    const selectedColor = propSelectedColor ?? getDefaultColor();
+
+    let colorNames: ReadonlyArray<Readonly<ColorNameItem>>;
+    if (brand === "workleap") {
```

Comment:
I would add a return before the if

Reply:
not sure I understand, a return for what?

Reply:
new line